makefiles/mcuboot.mk

```diff
@@ -1,16 +1,22 @@
+.PHONY: mcuboot mcuboot-create-key mcuboot-flash-bootloader mcuboot-flash
+
 ifdef MCUBOOT_SLOT0_SIZE
 
-IMGTOOL ?= $(RIOTTOOLS)/mcuboot/imgtool.py
-override IMGTOOL := $(abspath $(IMGTOOL))
+IMGTOOL ?= $(abspath $(RIOTTOOLS)/mcuboot/imgtool.py)
```

Comment:
Is the abspath even necessary?

Reply:
RIOTTOOLS is already an absolute path, so should not.

Reply:
```suggestion
IMGTOOL ?= $(RIOTTOOLS)/mcuboot/imgtool.py
```

Reply:
I think yes, will need to be re-tested with everything.